vendor/bat-native-ledger/src/bat/ledger/internal/database/database_server_publisher_banner.cc

```diff
@@ -164,6 +159,9 @@ void DatabaseServerPublisherBanner::OnGetRecord(
           banner,
           callback);
   links_->GetRecord(publisher_key, links_callback);
+
+  auto banner_pointer = type::PublisherBanner::New(banner);
+  callback(std::move(banner_pointer));
```

Comment:
Should this be in the `OnGetRecordLinks` method?

Reply:
Yep, I think so, since that's where the `amounts_->GetRecord()` call is missing now.

Reply:
Yes, thanks for catching! Fixed.